app/ui/FormCheckRadio.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactElement } from "react";
+import { Form } from "react-bootstrap";
+
+export function FormCheckRadio(props: {
+  onChange: (newUnit: string) => void;
+
+  unit: string;
+  value: string;
+}): ReactElement {
+  const { unit, value, onChange } = props;
+  return (
+    <Form.Check
+      type="radio"
+      label={unit}
+      checked={value === unit}
+      onClick={() => {
+        onChange(unit);
+      }}
+    ></Form.Check>
```

Comment:
empty tags should be self-closing

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/30/commits/c87002b34055111a4fcd66197fbdcad9c5522a9f